src/components/loading/_loading_content.scss

```
@@ -0,0 +1,36 @@
+$gradientStartStop: shadeOrTint($euiColorLightestShade, 8%, 12%);
+$gradientMiddle: shadeOrTint($euiColorLightestShade, 2%, 4%);
+
+.euiLoadingContent__loader {
+  width: 100%;
+}
+
+.euiLoadingContent__single-line {
```

Be sure to keep with the snake-case for the children as we use single dashes to denote states.
```suggestion
.euiLoadingContent__singleLine {
```